lib/SlackEventHandler.js

```diff
@@ -161,7 +177,28 @@ SlackEventHandler.prototype.handleMessageEvent = function (params) {
         channel_id: params.event.channel
     });
 
-    const processMsg = msg.text || msg.subtype === 'message_deleted' || msg.files != undefined || msg.subtype === 'message_changed';
+    // Handle events with attachments like bot messages.
+    if (params.event.type === "message" && params.event.attachments) {
+        for (var attachment of params.event.attachments) {
+            msg.text = attachment.fallback;
+            msg = await this.doChannelUserReplacements(msg, token);
+            await room.onSlackMessage(msg);
+        }
+        if (params.event.text == '') {
+            return;
+        }
+        msg.text = params.event.text;
+    }
+
+    const processMsg = msg.text || msg.files != undefined || msg.subtype === 'message_changed';
+
+    if (!processMsg) {
+        // TODO(paul): When I started looking at this code there was no lookupAndSendMessage()
+        //   I wonder if this code path never gets called...?
+        // lookupAndSendMessage(params.channel_id, params.timestamp, intent, roomID, token);
+        log.warn("Did not understand event: ", JSON.stringify(message));
```

Comment:
Shouldn't need to stringify, logger does that automatically.

Also make the `message` get logged as debug, but the error as warn because otherwise the logs will be incredibly bloated.

Reply:
Should `message` to be `msg` or `params` instead?